lib/modules/src/native/fs.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    callBridgeMethod,
+    callBridgeMethodSync,
+} from '@revenge-mod/modules/native'
+
+export function readFile(path: string) {
+    return callBridgeMethod('revenge.fs.read', [path])
+}
+
+export function writeFile(path: string, data: string) {
+    return callBridgeMethod('revenge.fs.write', [path, data])
+}
+
+export function exists(path: string) {
+    return callBridgeMethod('revenge.fs.exists', [path])
+}
+
+export function deleteFile(path: string) {
```

Comment:
maybe rename `deleteFile` and `deleteFileSync` to `rm` and `rmSync` so it's more similiar to `node:fs`? question mark?